CatProducer/prod/PAT2CAT_cfg.py

```diff
@@ -65,6 +65,22 @@
     
 process.catOutpath = cms.EndPath(process.catOut)    
 process.schedule.append(process.catOutpath)
+
+
+#### pileup weight
+import CATTools.CatProducer.catDefinitions_cfi as cat
+lumiJSON = "Cert_246908-260627_13TeV_PromptReco_Collisions15_25ns_JSON"
```

Comment:
i prefer to have the lumiJSON defined in catDefinitions_cfi, so all production info is in one place